terminus/builders/osm_city_builder.py

```diff
@@ -37,35 +37,30 @@ def get_city(self):
         # Get origin of the map (we use the center)
         tree = ET.parse(self.osm_map)
         bounds = tree.find('bounds').attrib
-        self.bounds = {'minlat': float(bounds['minlat']),
-                       'maxlat': float(bounds['maxlat']),
-                       'minlon': float(bounds['minlon']),
-                       'maxlon': float(bounds['maxlon'])}
+        self.bounds = {'origin': LatLon(float(bounds['minlat']), float(bounds['minlon'])),
+                       'corner': LatLon(float(bounds['maxlat']), float(bounds['maxlon']))}
 
-        self.map_origin = LatLon((self.bounds['minlat'] + self.bounds['maxlat']) / 2,
-                                 (self.bounds['minlon'] + self.bounds['maxlon']) / 2)
+        self.map_origin = self.bounds['origin'].middle_point(self.bounds['corner'])
```

Comment:
I'm not supper happy with the `middle_point` name, but I can't think of anything better now. Lets keep that in the back of our heads in case we find some inspiration in the future.

https://martinfowler.com/bliki/TwoHardThings.html
https://www.quora.com/Why-is-naming-things-hard-in-computer-science-and-how-can-it-can-be-made-easier